src/message-box.scss

```diff
@@ -27,14 +27,25 @@ $message-box-states: (
 .#{$block} {
   @include fd-reset();
 
+  position: relative;
+  z-index: 1;
```

Comment:
I don't think if it's good idea to put it there, can you add it to `fd-message-box-docs-static` this class?